client/src/components/signup/SignUp.js

```diff
@@ -71,6 +71,7 @@ class SignUp extends Component {
         let username = e.target.children[0].children[1].value;
         let email = e.target.children[1].children[1].value;
         let password = e.target.children[2].children[1].value;
+        let cryptoProfile= e.target.children[3].children[1].selectedOption;
```

Comment:
Does cryptoProfile on line 74 contain just one selectedOption or does it contain multiple selected options if the user has selected multiple cryptocurrencies.

Reply:
This doesn't work, yet. Kind of pseudo-coded it, for now.  Still trying to figure out how to insert all selections in the db. I think I figured it out, but I'll continue it on another day.